docs/docs/en/guide/task/stored-procedure.md

```diff
@@ -1,4 +1,10 @@
-# Stored Procedure
+# PROCEDURE
+
+## Overview
+
+`PROCEDURE` task type, the program used to execute the stored procedure
+
+## Create Task
 
 - Execute the stored procedure according to the selected DataSource.
```

Comment:
We should also add how to create task like others `- Drag from the toolbar <img src="/img/tasks/icons/procedure.png" width="25"/> to the canvas.`